util/pyclient/bft_msgs.py

```diff
@@ -47,18 +48,21 @@ def __init__(self, message):
 ReplyHeader = namedtuple('ReplyHeader', ['primary_id',
     'req_seq_num', 'length'])
 
-def pack_request(client_id, req_seq_num, read_only, msg):
+def pack_request(client_id, req_seq_num, read_only, msg, pre_process=False):
     """Create and return a buffer with a header and message"""
     flags = 0
     if read_only:
         flags = 1
     header = RequestHeader(client_id, flags, req_seq_num, len(msg))
-    data = b''.join([pack_request_header(header), msg])
+    data = b''.join([pack_request_header(header, pre_process), msg])
     return data
 
-def pack_request_header(header):
+def pack_request_header(header, pre_process=False):
```

Comment:
Please combine all request flags together (read_only, pre_process) to 'flags' field.